experiment/compatibility-versions/e2e-k8s-compatibility-versions.sh

```diff
@@ -70,8 +70,8 @@ trap signal_handler INT TERM
 
 # build kubernetes / node image, e2e binaries
 build() {
-  # build the node image w/ kubernetes
-  kind build node-image -v 1
+  # build the node image w/ kubernetes  (NOT USED FOR INITIAL CLUSTER, but for upgrade)
```

Comment:
if we're not using the image we can just build kubectl / kubelet right? that should be faster

Reply:
(not blocking, but I'd suggest considering this, compiling enough of k/k to create a working cluster fully is pretty slow)

Reply:
(also for at least one of the versions we should fetch CI binaries instead of compiling it at all ...)

Reply:
... actually it's not clear what we're using this for, is it just to build the tests / test tools now? it might be clearer to drop this comment and reflect that we're just building for testing?

Reply:
Yes exactly this just builds the test and test tools now (that comment was incorrect, thanks for flagging it).  I've removed this comment and updated the function comment to explain we are just building for testing